examples/glTFViewer/glTFViewer.cpp

```diff
@@ -252,10 +254,24 @@ namespace {
             .SetBindingsType(nxt::ShaderStageBit::Fragment, nxt::BindingType::SampledTexture, 2, 1)
             .GetResult();
 
+        auto renderpass = device.CreateRenderPassBuilder()
+            .SetAttachmentCount(1)
+            .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+            .SetSubpassCount(1)
+            .SubpassSetColorAttachment(0, 0, 0)
+            .GetResult();
+        auto framebuffer = device.CreateFramebufferBuilder()
+            .SetRenderPass(renderpass)
+            // attachment 0 -> back buffer
+            // (implicit) // TODO(kainino@chromium.org): use the texture provided by WSI
+            .SetDimensions(640, 480)
```

Comment:
MaterialInfo is a struct I have in the glTFViewer. If we wanted to be able to resize the viewport, the materials would all have to be recompiled, basically, but that seems right. glTF 2.0 probably solves this, anyway (just one pipeline by default).